### PR TITLE
queues: clarify consumer best practices

### PR DESCRIPTION
**Pull request by elithrar**

### CODE REVIEW

File: content/queues/learning/how-queues-works.md [hunk 1]
```diff
@@ -74,6 +76,12 @@ filename: wrangler.toml
 
 Importantly, each queue can only have one active consumer. This allows Cloudflare Queues to achieve at least once delivery and minimize the risk of duplicate messages beyond that.
 
+{{<Aside type="note">}}
+
+**Best practice**: configure a single consumer per queue. This both logically separates your queues, and ensures that errors (failures) in processing messages from one queue do not impact your other queues.
+
```
Comment (deadlypants1973) [inline content/queues/learning/how-queues-works.md]:
```suggestion
{{<Aside type="note" header="Best practice">}}

Configure a single consumer per queue. This both logically separates your queues, and ensures that errors (failures) in processing messages from one queue do not impact your other queues.

```

File: content/queues/learning/how-queues-works.md [hunk 2]
```diff
@@ -44,7 +44,9 @@ export default {
 };
 ```
 
-A queue can have multiple producers. For example, you may have multiple Workers writing events or logs to a shared queue based on incoming HTTP requests from users. There is no limit to the total number of producer Workers that can write to a single queue. Additionally, multiple queues can be bound to a single Worker. That single Worker can decide which queue to write to (or write to multiple) based on any logic you define in your code.
+A queue can have multiple producers. For example, you may have multiple Workers writing events or logs to a shared queue based on incoming HTTP requests from users. There is no limit to the total number of producer Workers that can write to a single queue.
```
Comment (deadlypants1973) [inline content/queues/learning/how-queues-works.md]:
```suggestion
A queue can have multiple producer Workers. For example, you may have multiple producer Workers writing events or logs to a shared queue based on incoming HTTP requests from users. There is no limit to the total number of producer Workers that can write to a single queue.
```

File: content/queues/learning/how-queues-works.md [hunk 3]
```diff
@@ -44,7 +44,9 @@ export default {
...
+
+Additionally, multiple queues can be bound to a single Worker. That single Worker can decide which queue to write to (or write to multiple) based on any logic you define in your code.
```
Comment (deadlypants1973) [inline content/queues/learning/how-queues-works.md:49]: Is this Worker just a regular Worker? Or do we mean 'single producer Worker' here?

**Review by deadlypants1973 — APPROVED** [review]
Approved with comment

File: content/queues/learning/how-queues-works.md (hunk 3 above)
Comment (elithrar) [reply content/queues/learning/how-queues-works.md:49]: Any Worker can write to a Queue, so "just a regular Worker" applies here.

**Pull request merged**